src/Orleans/Configuration/ClusterConfiguration.cs

```diff
@@ -441,7 +441,7 @@ internal static IPAddress ResolveIPAddress(string addrOrHost, byte[] subnet, Add
                 }
 
                 var candidates = new List<IPAddress>();
-                IPAddress[] nodeIps = Dns.GetHostAddresses(addrOrHost);
+                IPAddress[] nodeIps = Dns.GetHostAddressesAsync(addrOrHost).GetAwaiter().GetResult();
```

Comment:
Because of https://github.com/aspnet/Security/issues/59, and current callers of ResolveIPAddress wouldn't benefit  much from changing it to async (seems like the is only one who could gain from it is UpdateConfiguration method in SiloControl) so I haven't made such change initially.